tools/cldr-code/src/main/java/org/unicode/cldr/util/UnitConverter.java

```diff
@@ -241,23 +241,113 @@ public UnitConverter cloneAsThawed() {
     public static final class ConversionInfo implements Comparable<ConversionInfo> {
         public final Rational factor;
         public final Rational offset;
+        public String special;
+        public boolean inverse; // only used with special
```

Comment:
if this were specialInverse the meaning would always be clear.
```suggestion
        public boolean specialInverse; // only used with special
```